### DISCUSSION TITLE
Maintained

### DISCUSSION THREAD

**Discussion by adamzvolanek** [keydrive/keydrive#139] [General]
Does this continue to be maintained?

**Comment by ChappIO** [comment]
Hi @adamzvolanek we aren't currently working on any new features but if you run into any issues we'll be more than happy to take a look.

**Reply by adamzvolanek** [reply]
Sounds good, I'll give this a try! 

Thanks!

**Reply by LucaScorpion** [reply]
Great, do let us know what you think! I still actively use it as well, and overall I believe it should be pretty usable :)

**Comment by adamzvolanek** [comment]
Overall installation went smoothly, I appreciate the doker-compose file being included in the readme and ready to go. I use(d) Unraid to start this and noticed that this a basic cloud file sharing utility. 

I think your open issues in terms of improvements are a good map to continue if there is desire to do so. I didn't give this as much time as I'd like but wanted to reply saying I got it working and looks to be in working order. 

Do you want me to look into adding this into the Unraid Community apps as a template so it is available for more people?

**Reply by LucaScorpion** [reply]
Good to hear! I also use Unraid, and adding it to CA is definitely somewhere on my list of stuff I'd want to do :sweat_smile: But if you want to look into that, that would be amazing! Alternatively, it might be a good starting point to create a template repository, like https://github.com/limetech/docker-templates, since I've got no clue how much effort it is to get something into CA. Either way, really appreciate the effort :) And let me know if you need help with anything!
